### DISCUSSION TITLE
FQDN Expiring

### DISCUSSION THREAD

**Discussion by hwalker928** [pterodactyl/panel#2909] [Support] [answered]
When my domain for my FQDN of my node expires, what do I do? How can I change it to another domain?

**Comment by DaneEveritt** [comment]
Just edit it in the node settings to be the new one?

**Comment by FoksVHox** [comment, marked as answer]
You're able to change the FQDN by going under the node's settings and change the FQDN, after that you need to change the Wings' configuration.
